### ISSUE TITLE
Unable to show empty radar

### ISSUE THREAD

**Issue by bigyank** [imaNNeo/fl_chart#1217]
**Describe the bug**
The radar chart throws error if all the values in data entry is zero.

**To Reproduce**
```dart
import 'package:flutter/material.dart';
import 'package:fl_chart/fl_chart.dart';

const Color darkBlue = Color.fromARGB(255, 18, 32, 47);

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      theme: ThemeData.dark().copyWith(
        scaffoldBackgroundColor: darkBlue,
      ),
      debugShowCheckedModeBanner: false,
      home: Scaffold(
        body: Center(
          child: ProfileRadarChart(),
        ),
      ),
    );
  }
}

/// Widget [ProfileRadarChart] : Widget for Profile Chart
class ProfileRadarChart extends StatefulWidget {
  const ProfileRadarChart({super.key});

  @override
  State<ProfileRadarChart> createState() => _ProfileRadarChartState();
}

class _ProfileRadarChartState extends State<ProfileRadarChart> {
  /// FUNC [_convertScoreToList] : convert each score to double
  List<double> _convertScoreToList() {
    // throws an error if all the values are zero
    // doesn't throw an error if one of the value is non-zero
    return [
      double.parse('0'),
      double.parse('0'),
      double.parse('0'),
      double.parse('0'),
      double.parse('0'),
    ];
  }

  /// FUNC [_setTitleList] : set score title
  List<String> _setTitleList() {
    return [
      'followers',
      'posts',
      'applications',
      'winnings',
      'credibility',
    ];
  }

  /// FUNC [_buildDataSets] : Build Data sets for each entries
  List<RadarDataSet> _buildDataSets() {
    return [
      RadarDataSet(
        fillColor: Colors.black12,
        borderColor: Colors.blueAccent,
        entryRadius: 0,
        dataEntries:
            _convertScoreToList().map((e) => RadarEntry(value: e)).toList(),
        borderWidth: 2,
      )
    ];
  }

  @override
  Widget build(BuildContext context) {
    return AspectRatio(
      aspectRatio: 1.5,
      child: RadarChart(
        RadarChartData(
          dataSets: _buildDataSets(),
          radarShape: RadarShape.circle,
          radarBorderData: const BorderSide(color: Colors.black12, width: 0),
          radarBackgroundColor: Colors.white,
          borderData: FlBorderData(show: false),
          titlePositionPercentageOffset: 0.15,
          titleTextStyle: const TextStyle(
            color: Colors.black,
            fontSize: 10,
          ),
          getTitle: (index, angle) {
            return RadarChartTitle(
              text: _setTitleList()[index],
            );
          },
          tickCount: 4,
          ticksTextStyle:
              const TextStyle(color: Colors.transparent, fontSize: 10),
          tickBorderData: const BorderSide(color: Colors.black12, width: 0),
        ),
      ),
    );
  }
}
```

**Screenshots**
| Chart      | Error |
| ----------- | ----------- |
| <img src="https://i.imgur.com/YMAXyZ2.png" width="300" />     |<img src="https://i.imgur.com/zCXuS90.png" width="300" />       |

**Versions**
 - Flutter : Flutter 3.3.10
 - FlChart :   fl_chart: ^0.55.2

**Comment by imaNNeo** [comment]
Please provide a reproducible code.
Thanks

**Comment by bigyank** [comment]
@imaNNeo I've updated the issue with reproducible code.

**Comment by imaNNeo** [comment]
OKay now I can reproduce it.

**Comment by imaNNeo** [comment]
Fixed in [0.60.0](https://github.com/imaNNeo/fl_chart/releases/tag/0.60.0). Please check it out!
